tests/langchain_chat_sample.py

```diff
@@ -154,17 +118,32 @@ def format_docs(docs):
 #         "status_code": "UNSET"
 #     },
 #     "attributes": {
-#         "session.session_id": "0x4fa6d91d1f2a4bdbb7a1287d90ec4a16"
+#         "session.session_id": "0x4fa6d91d1f2a4bdbb7a1287d90ec4a16",
+#         "tags": [
+#             "Chroma",
+#             "OpenAIEmbeddings"
+#         ],
+#         "type": "vector_store",
+#         "provider_name": "OpenAIEmbeddings",
+#         "embedding_model": "Chroma"
```

Comment:
provider_name is Chroma
and embedding_model is OpenAIEmbeddings
please check the code

Reply:
CC: @akshay-okahu  pls review the attribute names

Reply:
done @oi-raanne